### PR TITLE
Build nextalign for osx-arm64

### PR DESCRIPTION
**Pull request by corneliusroemer**

### CODE REVIEW

**Comment by corneliusroemer** [comment]
@BiocondaBot please add label

**Comment by corneliusroemer** [comment]
@BiocondaBot please fetch artifacts

File: recipes/nextalign/meta.yaml [hunk 1]
```diff
@@ -49,3 +50,6 @@ extra:
   skip-lints:
     # repackaged binary
     - should_be_noarch_generic
+  identifiers:
+    - doi:10.21105/joss.03773
+    - doi:10.21105/joss.03773
```
Comment (mbhall88) [inline recipes/nextalign/meta.yaml:55]:
```suggestion
```

Comment (mbhall88) [reply recipes/nextalign/meta.yaml:55]: seems there is a doubleup of the DOI

Comment (corneliusroemer) [reply recipes/nextalign/meta.yaml]: ah woops, not sure how that happened. Thanks for the catch!

**Review by martin-g — APPROVED** [review]

**Pull request merged**